lib/i18n_rails_helpers/model_helpers.rb

```diff
@@ -0,0 +1,45 @@
+module I18nRailsHelpers
+  module ModelHelpers
+    # enum attrubute_t and attributes_t return translated enum values
+    #
+    # Example:
+    #   in the Client model
+    #     enum gender: { undefined: 0, female: 1, male: 2 }
+    #   in use
+    #     Client.first.gender    # => 'female'
+    #     Client.first.gender_t  # => 'Frau'
+    #     Client.first.genders_t # => { undefined: 'Nicht definiert', female: 'Frau', male: 'Mann' }
+    #
+    # Requires:
+    #   locale key: activerecord.attributes.#{model_name}.#{enum}.#{enum_value_key}
+    #   eg.:        activerecord.attributes.client.gender.female                      # => 'Frau'
```

Comment:
I think the key is plural here, i.e. "genders"  (`#{enum_attr.to_s.pluralize}` below)

Reply:
true!